ci/scripts/manylinux_build_wheel.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -ex
+
+# Parse ARGs
+for ARGUMENT in "$@"
+do
+    KEY=$(echo $ARGUMENT | cut -f1 -d=)
+    VALUE=$(echo $ARGUMENT | cut -f2 -d=)
+    case "$KEY" in
+            SHA)              SHA=${VALUE} ;;
+            PY)               PY=${VALUE} ;;
+            *)
+    esac
+done
+
+source /home/dev/miniconda/etc/profile.d/conda.sh
+conda activate $PY
+
+# Build Taichi from source
+git clone --recursive https://github.com/taichi-dev/taichi --branch=master
+cd taichi
+git checkout $SHA
+python3 -m pip install -r requirements_dev.txt -i http://repo.taichigraphics.com/repository/pypi/simple --trusted-host repo.taichigraphics.com
+TAICHI_CMAKE_ARGS="-DTI_WITH_VULKAN:BOOL=OFF -DTI_WITH_CUDA:BOOL=OFF -DTI_WITH_OPENGL:BOOL=OFF -DTI_WITH_CC:BOOL=OFF" python3 setup.py install
+
+# Add Docker specific ENV
+export TI_IN_DOCKER=true
+
+# Run tests
+ti diagnose
+
+# Build wheel
+cd python && python build.py build
```

Comment:
This performs build twice(the first at Line 25). And `build.py` is to be removed by #3537 .

Maybe we can modify Line 25 to build the wheel and install it in two steps.

Reply:
sure, we can stay aligned here.